sys/src/libufs/type.c

```diff
@@ -25,28 +25,27 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include <sys/cdefs.h>
-__FBSDID("$FreeBSD$");
-
-#include <sys/param.h>
-#include <sys/mount.h>
-#include <sys/disklabel.h>
-#include <sys/stat.h>
-
-#include <ufs/ufs/ufsmount.h>
-#include <ufs/ufs/dinode.h>
-#include <ufs/ffs/fs.h>
-
-#include <errno.h>
-#include <fcntl.h>
-#include <fstab.h>
-#include <paths.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <unistd.h>
-
-#include <libufs.h>
+//#include <sys/cdefs.h>
+
+//#include <sys/param.h>
+//#include <sys/mount.h>
+//#include <sys/disklabel.h>
+//#include <sys/stat.h>
+
+#include <ufs/ufsmount.h>
+#include <ufs/dinode.h>
+#include <ffs/fs.h>
+
+//#include <errno.h>
+//#include <fcntl.h>
+//#include <fstab.h>
+//#include <paths.h>
+//#include <stdio.h>
+//#include <stdlib.h>
+//#include <string.h>
+//#include <unistd.h>
+
+#include <lib/libufs.h>
```

Comment:
lib/libufs.h here

this is a bit confusing. DO they need to be separate (maybe they do)?)

Reply:
Nope, typo.  Was meant to be <ufs/libufs.h>.